pkgs/development/python-modules/stups/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv
```

unless specifically referencing something off stdenv, it's prefered to leave it out. if you need lib, then just import that directly.
```suggestion
{ lib
```